nixos/modules/services/misc/libreddit.nix

```diff
@@ -56,6 +56,31 @@ in
           AmbientCapabilities = lib.mkIf (cfg.port < 1024) [ "CAP_NET_BIND_SERVICE" ];
           Restart = "on-failure";
           RestartSec = "2s";
+          # Hardening
+          CapabilityBoundingSet = if (cfg.port < 1024) then [ "CAP_NET_BIND_SERVICE" ] else [ "" ];
```

Comment:
```suggestion
          CapabilityBoundingSet = lib.optional (cfg.port < 1024) "CAP_NET_BIND_SERVICE";
```

Reply:
> If this option is not used, the capability bounding set is not modified on process execution, hence no limits on the capabilities of the process are enforced.
>
> -- https://www.freedesktop.org/software/systemd/man/systemd.exec.html#Capabilities

I believe that with `lib.optional`, the option will not be defined and capabilities will not be changed.